helm-frontend/src/components/NavDropdown.tsx

```diff
@@ -18,10 +18,21 @@ function NavDropdown() {
         setProjectMetadata(data);
         // set currProjectMetadata to val where projectMetadataEntry.id matches window.PROJECT_ID
         if (window.PROJECT_ID) {
-          const currentEntry = data.find(
-            (entry) => entry.id === window.PROJECT_ID,
-          );
-          setCurrProjectMetadata(currentEntry);
+          if (window.PROJECT_ID === "global") {
```

Comment:
seems a bit weird, but okay for now

Reply:
I will make a PR to fix this once we update the project metadata to have global (and image_urls)